dbt/models/default/docs.md

```diff
@@ -170,7 +177,7 @@ process. The full data lineage looks something like:
 
 ![Data Flow Diagram](./assets/sales-lineage.svg)
 
-**Primary Key**: `doc_no`, `pin`
+**Primary Key**: `doc_no`, `pin`, `sale_price`
```

Comment:
This is not exactly correct since it leaves out the `sale_filter_same_sale_within_365` exclusion, but it's close enough, and hopefully the expanded "Nuance" section will provide the necessary context.